portfolio/src/main/webapp/script.js

```diff
@@ -66,15 +66,28 @@ function onBodyLoad() {
 }
 
 /**
- * Fetch comments array from the server and load them to the list of comments, if it exists.
+ * Clear comments, then fetch them from the server and 
+ * load them to the list of comments, if it exists.
  */
 function loadComments() {
-  fetch("/data").then(response => response.json()).then(comments => {
+  clearComments();
+  let numComments = document.getElementById("num-comments")?.getAttribute("value");
+
+  fetch("/data?num-comments=" + numComments).then(response => response.json()).then(comments => {
     const commentsEl = document.getElementById("comments-list");
     comments.forEach(comment => {
       const liElement = document.createElement('li');
       liElement.innerText = comment;
       commentsEl?.appendChild(liElement);
     });
   });
+}
+
+/**
+ * Clear comments from comments list.
+ */
+function clearComments() {
+  const commentsEl = document.getElementById("comments-list");
+  if (commentsEl)
```

Comment:
You shouldn't imply curly braces. Throw them in here

Reply:
Gotcha. Is `  if (commentsEl) commentsEl.innerHTML = "";` Ok?

Reply:
Yes, see here for more info about google's style guide:
go/tsstyle#control-flow-statements-blocks

Reply:
Got hit with UberProxy 🤔 but I found this one https://google.github.io/styleguide/jsguide.html

Reply:
Ah weird. Here is the relevant bit in the guide you linked:

go/tsstyle#control-flow-statements-blocks